examples/disaggregated/slurm/benchmark/submit.py

```
@@ -111,6 +115,7 @@ def submit_job(config):
         f'--ntasks={total_tasks}',
         f'--ntasks-per-node={hw_config["gpus_per_node"]}',
         f'--segment={total_nodes}',
+        f'--reservation=oos_fix_gdrdrv_dkms',
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)